Gruntfile.coffee

```diff
@@ -1227,11 +1208,15 @@ module.exports = (grunt) ->
     'jasmine_chromeapp:socksEcho'
   ]
 
+  grunt.registerTask 'unit_test_nobuild', _.flatten(
```

Comment:
FYI, I'm fixing it with: https://github.com/uProxy/uproxy/commit/340fde065ddf15004879c47847d6908d12ba602f

Unfortunately two tests are now broken: #2727. I hope the release is not broken.